### PR TITLE
Fix to handleKeyPress in ClickableComponent

### PR DESCRIPTION
**Pull request by marguinbc**
## Description
Patch ClickableComponent but adding check to see if component is a DIV and not a button to determine if enter and space should be handled.

## Specific Changes proposed
Added condition to check if element is a DIV

## Requirements Checklist
- [x ] Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](http://jsbin.com/axedog/edit?html,output))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/js/button.js [hunk 1]
```diff
@@ -87,9 +87,9 @@ class Button extends ClickableComponent {
    * @method handleKeyPress
    */
   handleKeyPress(event) {
+    // If the element is a div acting like a button, use super's handleKeyPress but 
     // Ignore Space (32) or Enter (13) key operation, which is handled by the browser for a button.
-    if (event.which === 32 || event.which === 13) {
-    } else {
+    if ((this.el_.tagName.toUpperCase() === 'DIV') && ((event.which !== 32) || (event.which !== 13))) {
```
Comment (OwenEdwards) [inline src/js/button.js]: This needs to work if the element is a <span>, or anything else that isn't a <button>; why not use the same logic for the test as in line 40?

Also, `((event.which !== 32) || (event.which !== 13))` will always evaluate to true, even if which is 32 or 13!!

**Comment by OwenEdwards** [comment]
The title and description of this PR are confusing because the "bug" is not in `ClickableComponent`, it's in `Button`. The whole point of splitting a Clickable Component from a Button was to fix a bug with Button handling, and the necessary changes were made elsewhere in video.js; this "bug" only shows up in the backward compatibility of the `Button`.

If we knew more about the other items which are extending `Button`, we could decide if we should actually honor their `<div>` passed as the `tag` parameter, or just ignore it and use a `<button>` element instead - that was a major goal in moving to V5. What do you think, @gkatsev & @heff?

**Comment by gkatsev** [comment]
I think this may belong elsewhere, going to talk with @marguinbc offline.

File: src/js/button.js (hunk 1 above)
Comment (marguinbc) [inline src/js/button.js]: Good point about div vs span and using the test on line 40.  The conditional logic was a brain fart, wanted to reverse the logic from 'do nothing if' and everything 'else' to 'do everything if' without an else.

**Pull request closed without merging**